src/api/index.tsx

```diff
@@ -102,9 +104,10 @@ export enum HttpMethod {
   PUT = 'PUT',
 }
 
-type CallApiParams = {
+type CallApiParams<BodyDataType extends void | {}> = {
   _makeQueryString?: typeof makeQueryString;
   apiState: ApiState;
+  bodyData?: BodyDataType;
```

Comment:
So this is saying that `bodyData` is either `void` or an object?

Reply:
Yeah. For GET requests, the body is always undefined (in our API's implementation). We send parameters instead.